langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -707,6 +707,72 @@ function testShiftOnTupleWithoutValuesForRestParameter() {
     assertValueEquality("shift() not supported on type '[int,int...]'", detailMessage);
 }
 
+function testUnshiftOperation() {
```

Comment:
Shall we improve the test cases with structured types like Records as well?

Reply:
Thank you for the suggestion. Added with 1ce03d1d06b73799e855ee1ad2bd76aaeee937d1